NetappReplication/cli.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2019 The OpenSDS Authors.
```

Comment:
Any particular reason not to use the API for snap mirror?

Reply:
Yes. currently vendors package of netapp also doesn't have API for snap mirror.